packages/date-picker/test/keyboard-input.test.js

```diff
@@ -257,70 +256,43 @@ describe('keyboard', () => {
     });
 
     describe('focus date not in the viewport', () => {
-      let spy;
-
-      afterEach(() => {
-        spy.restore();
-      });
-
-      it('should focus date scrolled out of the view on input Tab', async () => {
-        // Move focus to the calendar
-        await sendKeys({ press: 'Tab' });
-        await nextRender(datepicker);
+      beforeEach(async () => {
+        // Disable scrolling animation
+        const stub = sinon.stub(overlayContent, 'revealDate').callsFake((date) => {
```

Comment:
We might try to set `scrollDuration: 0` on the overlay content instead, not sure which is better.